spec/classes/init_spec.rb

```diff
@@ -126,14 +130,15 @@
     'Suse-12-x86_64' => {
       :architecture           => 'x86_64',
       :osfamily               => 'Suse',
+      :operatingsystem        => 'SLES',
       :operatingsystemrelease => '12.0',
       :ssh_version            => 'OpenSSH_6.6.1p1',
       :ssh_version_numeric    => '6.6',
       :ssh_packages           => ['openssh'],
       :sshd_config_mode       => '0600',
       :sshd_service_name      => 'sshd',
       :sshd_service_hasstatus => true,
-      :sshd_config_fixture    => 'sshd_config_suse_x86_64',
+      :sshd_config_fixture    => 'sshd_config_suse_i386',
```

Comment:
@ghoneycutt 

Can we create a new fixture file sshd_config_suse12_x86_64 for sles 12 , which will have "Subsystem sftp /usr/lib/ssh/sftp-server" to match the line 145 in init.pp  ?